### PR TITLE
Backend: League of Legends Actions

### PR DESCRIPTION
**Pull request by EdenComp**
Also fixes ARE-399
Also fixes ARE-403
Also fixes ARE-404
Also fixes ARE-405

### CODE REVIEW

File: backend/back/src/types/jobParams.ts [hunk 1]
```diff
@@ -223,3 +233,28 @@ export class FacebookPageParams {
...
+type RiotRegionType = keyof typeof RiotRegion;
+
+export class RiotActionsParams extends UniqueJobParams {
+	@IsEnum(RiotRegion)
+	region: RiotRegionType;
+
+	@IsString()
+	@MinLength(3)
+	@MaxLength(16)
+	summoner: string;
```
Comment (Croos3r) [inline backend/back/src/types/jobParams.ts:259]: Don't you have a list o sommner ? Could ask pablo for that

File: backend/back/src/workflows/seed/1698964479850-CreateRiotServiceAndActions.ts
Comment (Croos3r) [inline backend/back/src/workflows/seed/1698964479850-CreateRiotServiceAndActions.ts:1]: Please split this migration, service in src/services/seed and areas in src/workflows/seed

**Review by Croos3r — CHANGES_REQUESTED** [review]

File: backend/back/src/types/jobParams.ts (hunk 1 above)
Comment (EdenComp) [reply backend/back/src/types/jobParams.ts:259]: Summoner = a player (here I want the player username)
Champion = a playable character

File: backend/back/src/workflows/seed/1698964479850-CreateRiotServiceAndActions.ts
Comment (EdenComp) [reply backend/back/src/workflows/seed/1698964479850-CreateRiotServiceAndActions.ts:1]: Will do

File: backend/back/src/types/jobIds.ts [hunk 2]
```diff
@@ -62,5 +62,9 @@ export const JobsIdentifiers: JobsIdentifiers = {
 	"linear-on-issue-create": ({ workspace }) => uniqueWebhookId("linear-on-issue-create", workspace),
 	"linkedin-create-post": ({ workflowStepId }) => uniqueJobId("linkedin-create-post", workflowStepId),
 	"miro-create-board": ({ workflowStepId }) => uniqueJobId("miro-create-board", workflowStepId),
+	"riot-on-game-end": ({ region, summoner }) => `riot-on-game-end-${region}-${summoner}`,
+	"riot-on-game-loss": ({ region, summoner }) => `riot-on-game-loss-${region}-${summoner}`,
+	"riot-on-game-win": ({ region, summoner }) => `riot-on-game-win-${region}-${summoner}`,
+	"riot-on-level-up": ({ region, summoner }) => `riot-on-level-up-${region}-${summoner}`,
```
Comment (RezaRahemtola) [inline backend/back/src/types/jobIds.ts:68]: I would add `lol` somewhere, as it's not a Valorant game for example, which is also Riot

**Review by Croos3r — APPROVED** [review]
LGTM

**Review by RezaRahemtola — APPROVED** [review]

**Pull request merged**